backend-service/Embedded/RelayTest.py

```diff
@@ -0,0 +1,83 @@
+from RelayContainer import RelayContainer
+from Relay import Relay
+import unittest
+
+class RelayContainerTestCase(unittest.TestCase): 
+    def setUp(self) -> None:
+       #create a blank relay container
+       self.con = RelayContainer()
+    
+    def test_con_creation(self):
+        self.assertIsInstance(self.con, RelayContainer, "Created container is not instance of RelayContainer")
+
+    def test_add_relay(self):
+        prev_size = self.con.get_size()
+        self.con.add_relay(1, True)
```

Comment:
Need to add more parameters for the add_relay functions in order to make it compatible